controls/C-0222-minimizeuseraccesstoamazonecr.json

```diff
@@ -0,0 +1,21 @@
+    "manual_test": "",
+    "references": [
+        "https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-scanning.html#scanning-repository"
+    ],
+    "attributes": {
+        "armoBuiltin": true,
+        "actionRequired": "requires review"
```

Comment:
let's not add this attribute yet as it doesn't work properly, there's a [bug open for it](https://cyberarmor-io.atlassian.net/browse/SUB-665)

Reply:
So we should leave it blank, or we can use some alternative to the `requires review` value?